tests/framework/utils/helm_helper.go

```diff
@@ -0,0 +1,87 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/pkg/util/exec"
+)
+
+var hhlogger = capnslog.NewPackageLogger("github.com/rook/rook", "helmhelper")
+
+//HelmHelper is wrapper for running helm commands
+type HelmHelper struct {
+	executor *exec.CommandExecutor
+}
+
+//NewHelmHelper creates a instance of HelmHelper
+func NewHelmHelper() *HelmHelper {
+	executor := &exec.CommandExecutor{}
+	return &HelmHelper{executor: executor}
+
+}
+
+//Helm is wrapper for executing helm commands
+func (hh *HelmHelper) Helm(args ...string) (string, error) {
+	result, err := hh.executor.ExecuteCommandWithOutput(false, "", "helm", args...)
+	if err != nil {
+		hhlogger.Errorf("Errors Encountered while executing helm command : %v", err)
+		return "", fmt.Errorf("Failed to run helm commands on args %v : %v", args, err)
+
+	}
+	return result, nil
+
+}
+
+//GetLocalRookHelmChartVersion returns helm chart version for a give chart
+func (hh *HelmHelper) GetLocalRookHelmChartVersion(chartName string) (string, error) {
+	cmdArgs := []string{"search", chartName}
+	result, err := hh.Helm(cmdArgs...)
+	if err != nil {
+		hhlogger.Errorf("cannot find helm chart %v : %v", chartName, err)
+		return "", fmt.Errorf("Failed to find helm chart  %v : %v", chartName, err)
+	}
+
+	if strings.Contains(result, "No results found") {
+		return "", fmt.Errorf("Failed to find helm chart  %v ", chartName)
+	}
+
+	lines := strings.Split(result, "\n")
```

Comment:
instead of splitting on newline and tab, you might use the `Grep` and `Awk` helpers in our `pkg/util/sys` package.